packages/vite/src/node/server/moduleGraph.ts

```diff
@@ -179,6 +179,7 @@ export class ModuleGraph {
 
     const mod = new ModuleNode(url)
     mod.file = file
+
```

Comment:
Would you remove these added new lines? So this file is not part of the change set
```suggestion
```